lib/oauth.js

```diff
@@ -155,6 +165,15 @@ module.exports = new Model({
     ].join('');
   },
 
+  _bearerTokenIsExpired: function() {
+    var tokenDetails = JSON.parse(SESSION_STORAGE.getItem(LOCAL_STORAGE_PREFIX + 'tokenDetails'));
+    if (tokenDetails) {
+      return Date.now() >= tokenDetails.expires_at - TOKEN_EXPIRATION_ALLOWANCE;
```

Comment:
is 60s long enough to get a new token via an iFrame on slow network / a slow busy server?

Reply:
I don't know. What's a reasonable time to allow for Panoptes to respond?

Reply:
The load balancer kills connections that have not started responding after 60s so in theory it could be up to that long for a refresh promise to fulfill on a slow server. 

I prefer to be overly cautious on this as i'm pretty sure slow / late token refreshes have caused some auth issues with ASM. E.g. say if the refresh via iFrame setTimeout is 10s late to fire and the token refresh takes 55s then there is a 5s period where the token will be expired and invalid. 

Perhaps add 30s to it to be 180secs or maybe 2 mins?

Reply:
I've bumped it to 5 minutes. I've also been thinking that the solution to the unreliable timeout in #81 is to set the timer interval to the same allowance.